.travis.yml

```diff
@@ -5,12 +5,12 @@ jdk:
   - oraclejdk11
 
 install:
-   - cd src/test
+   - cd src/test/scripts
```

Comment:
Don't do the cd and back here!
Just directly call the scripts as mentioned in the installation instructions
```
source ./src/test/scripts/download_bigg_models.sh
source ./src/test/scripts/download_sbml-test-suite.sh
```

Reply:
Done!